src/index.js

```diff
@@ -46,11 +46,18 @@ class Radar {
     return STATUS;
   }
 
-  static initialize(publishableKey) {
+  static initialize(publishableKey, options={}) {
     if (!publishableKey) {
       console.error('Radar "initialize" was called without a publishable key');
     }
     Storage.setItem(Storage.PUBLISHABLE_KEY, publishableKey);
+    if(options){
+      let {locationTimeToLive} = options
```

Comment:
use `const` instead of `let` if it's not being reassigned.

Reply:
👍